lib/zmq-kernel.js

```diff
@@ -47,17 +47,10 @@ export default class ZMQKernel extends Kernel {
       });
 
       this.kernelProcess.stderr.on('data', (data) => {
-        data = data.toString();
-
-        log('ZMQKernel: stderr:', data);
-
-        const regexp = getKernelNotificationsRegExp();
-        if (regexp && regexp.test(data)) {
```

Comment:
LGTM

IPython and IJulia write to `stdout` on startup (so problem there).
If there are any kernels that write to `stderr` on startup, we can revisit this decision.
We'll also need to revisit this code, once Jupyter introduces the use of a kernel nanny.